### DISCUSSION TITLE
Welcome to 4d-google-workspace Discussions!

### DISCUSSION THREAD

**Discussion by macMikey** [macMikey/4d-google-workspace#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by pistou98** [comment]
hi Mikey,
My first question will be where to start in order to use google calendar.
I have a google key and now what's next ?

**Reply by pistou98** [reply]
I forgot I'm Carlos Pisterman

**Reply by macMikey** [reply]
I figured it was you, Carlos.
As we get this moving, please make sure that we get the docs updated so that they are more helpful to the next person who comes along.
I moved this conversation to a new topic, [Getting Started](https://github.com/macMikey/4d-google-workspace/discussions/16)
